bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java

```diff
@@ -0,0 +1,68 @@
+package emfprofiles.custom.wizards;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.sirius.business.api.componentization.ViewpointRegistry;
+import org.eclipse.sirius.business.api.helper.SiriusUtil;
+import org.eclipse.sirius.business.api.session.Session;
+import org.eclipse.sirius.ui.business.api.session.UserSession;
+import org.eclipse.sirius.viewpoint.description.RepresentationDescription;
+import org.eclipse.sirius.viewpoint.description.Viewpoint;
+
+public final class ViewPointUtil {
+
+    private ViewPointUtil() {
+        // intentionally left blank
+    }
+
+    public static Viewpoint getSelectedViewpointByName(Session session, String viewpointName) {
+        return session.getSelectedViewpoints(false)
+                .stream()
+                .filter(v -> viewpointName.equals(v.getName()))
+                .findAny()
+                .orElse(null);
+    }
+
+    public static RepresentationDescription findDescription(Viewpoint viewpoint, String descriptionName) {
+        return viewpoint.getOwnedRepresentations()
+                .stream()
+                .filter(d -> descriptionName.equals(d.getName()))
+                .findAny()
+                .orElse(null);
+    }
```

Comment:
Maybe I missed it, but I do not see any code using these methods.

Reply:
Correct, they are no longer used after other cleanup steps.